src/deepsparse/yolo/schemas.py

```diff
@@ -67,7 +67,7 @@ class YOLOOutput(BaseModel):
     scores: List[List[float]] = Field(
         description="List of scores, one for each prediction"
     )
-    labels: List[List[str]] = Field(
```

Comment:
We need to do some unification between the output schemas of yolact and yolo. In the next PR, I will set `labels` to `classes` so that the naming is consistent. In the future, I want to also make sure that `classes` in yolact and yolo actually expect the same data structure.